### PR TITLE
Add attributes to Header

### PR DESCRIPTION
**Pull request by tiltom**
https://taiga.sovryn.app/project/sovryn-light/task/407

### CODE REVIEW

**Review by creed-victor — APPROVED** [review]

File: src/app/components/Header/index.tsx [hunk 1]
```diff
@@ -49,50 +49,88 @@ export function Header() {
...
+    },
+    {
+      to: '/reward',
+      title: t(translations.mainMenu.reward),
+      dataAttribute: 'header-bitocracy-link-rewards',
+    },
+    {
+      to: '/wallet',
+      title: t(translations.mainMenu.wallet),
+      dataAttribute: 'header-portfolio-link-wallet',
```
Comment (tiltom) [inline src/app/components/Header/index.tsx:106]:
```suggestion
      dataAttribute: 'header-link-portfolio',
```

File: src/app/components/Header/index.tsx [hunk 2]
```diff
@@ -49,50 +49,88 @@ export function Header() {
...
     },
-    { to: '/stake', title: t(translations.mainMenu.staking) },
-    { to: '/reward', title: t(translations.mainMenu.reward) },
-    { to: '/wallet', title: t(translations.mainMenu.wallet) },
     {
       to: bridgeURL,
       title: t(translations.mainMenu.bridge),
+      dataAttribute: 'header-bridge',
```
Comment (tiltom) [inline src/app/components/Header/index.tsx:111]:
```suggestion
      dataAttribute: 'header-link-bridge',
```

File: src/app/components/Header/index.tsx [hunk 3]
```diff
@@ -49,50 +49,88 @@ export function Header() {
...
+      to: '/origins/claim',
+      title: t(translations.mainMenu.originsClaim),
+      dataAttribute: 'header-origins-link-claim',
     },
-    { to: '/origins', title: t(translations.mainMenu.origins) },
-    { to: '/origins/claim', title: t(translations.mainMenu.originsClaim) },
     {
       to: 'https://wiki.sovryn.app/en/sovryn-dapp/faq-dapp',
       title: t(translations.mainMenu.help),
+      dataAttribute: 'header-help',
```
Comment (tiltom) [inline src/app/components/Header/index.tsx:126]:
```suggestion
      dataAttribute: 'header-link-help',
```

File: src/app/components/Header/index.tsx [hunk 4]
```diff
@@ -49,50 +49,88 @@ export function Header() {
...
+    },
+    {
+      to: '/stake',
+      title: t(translations.mainMenu.staking),
+      dataAttribute: 'header-bitocracy-link-stake',
+    },
+    {
+      to: '/reward',
+      title: t(translations.mainMenu.reward),
+      dataAttribute: 'header-bitocracy-link-rewards',
```
Comment (soulBit) [inline src/app/components/Header/index.tsx:101]:
```suggestion
      dataAttribute: 'header-link-rewards',
```

File: src/app/components/Header/index.tsx [hunk 5]
```diff
@@ -199,27 +239,31 @@ export function Header() {
                       onClick={() => {
                         history.push('/buy-sov');
                       }}
+                      data-action-id="tradeMenu-buy-sov-btn"
```
Comment (soulBit) [inline src/app/components/Header/index.tsx:242]:
```suggestion
                      data-action-id="header-trade-link-buySov"
```

File: src/app/components/Header/index.tsx [hunk 6]
```diff
@@ -199,27 +239,31 @@ export function Header() {
...
                     />
                     <MenuItem
                       text={t(translations.mainMenu.swap)}
                       className="bp3-popover-dismiss"
                       onClick={() => {
                         history.push('/swap');
                       }}
+                      data-action-id="tradeMenu-swap-btn"
```
Comment (soulBit) [inline src/app/components/Header/index.tsx:250]:
```suggestion
                      data-action-id="header-trade-link-swap"
```

File: src/app/components/Header/index.tsx [hunk 7]
```diff
@@ -199,27 +239,31 @@ export function Header() {
...
                     />
                     <MenuItem
                       text={t(translations.mainMenu.spotTrade)}
                       className="bp3-popover-dismiss"
                       onClick={() => {
                         history.push('/spot');
                       }}
+                      data-action-id="tradeMenu-spot-btn"
```
Comment (soulBit) [inline src/app/components/Header/index.tsx:258]:
```suggestion
                      data-action-id="header-trade-link-spot"
```

File: src/app/components/Header/index.tsx [hunk 8]
```diff
@@ -199,27 +239,31 @@ export function Header() {
...
                     />
                     <MenuItem
                       text={t(translations.mainMenu.marginTrade)}
                       className="bp3-popover-dismiss"
                       onClick={() => {
                         history.push('/trade');
                       }}
+                      data-action-id="tradeMenu-margin-btn"
```
Comment (soulBit) [inline src/app/components/Header/index.tsx:266]:
```suggestion
                      data-action-id="header-trade-link-margin"
```

File: src/app/components/Header/index.tsx [hunk 9]
```diff
@@ -229,7 +273,10 @@ export function Header() {
                     isSectionOpen(SECTION_TYPE.TRADE) && 'tw-font-bold'
                   }`}
                 >
-                  <span className="tw-mr-2 2xl:tw-mr-3 tw-cursor-pointer">
+                  <span
+                    className="tw-mr-2 2xl:tw-mr-3 tw-cursor-pointer"
+                    data-action-id="headerMenu-trade-btn"
```
Comment (soulBit) [inline src/app/components/Header/index.tsx:278]:
```suggestion
                    data-action-id="header-link-trade"
```

File: src/app/components/Header/index.tsx [hunk 10]
```diff
@@ -242,16 +289,19 @@ export function Header() {
                       text={t(translations.mainMenu.lend)}
                       className="bp3-popover-dismiss"
                       onClick={() => history.push('/lend')}
+                      data-action-id="financeMenu-lend-btn"
```
Comment (soulBit) [inline src/app/components/Header/index.tsx:292]:
```suggestion
                      data-action-id="header-link-finance"
```

Comment (soulBit) [inline src/app/components/Header/index.tsx:292]:
```suggestion
                      data-action-id="header-finance-link-lend"
```

File: src/app/components/Header/index.tsx [hunk 11]
```diff
@@ -242,16 +289,19 @@ export function Header() {
...
                     />
                     <MenuItem
                       text={t(translations.mainMenu.borrow)}
                       className="bp3-popover-dismiss"
                       onClick={() => history.push('/borrow')}
+                      data-action-id="financeMenu-borrow-btn"
```
Comment (soulBit) [inline src/app/components/Header/index.tsx:298]:
```suggestion
                      data-action-id="header-finance-link-borrow"
```

File: src/app/components/Header/index.tsx [hunk 12]
```diff
@@ -242,16 +289,19 @@ export function Header() {
...
                     />
                     <MenuItem
                       text={t(translations.mainMenu.yieldFarm)}
                       className="bp3-popover-dismiss"
                       onClick={() => history.push('/yield-farm')}
+                      data-action-id="financeMenu-yieldFarm-btn"
```
Comment (soulBit) [inline src/app/components/Header/index.tsx:304]:
```suggestion
                      data-action-id="header-finance-link-yieldFarm"
```

File: src/app/components/Header/index.tsx [hunk 13]
```diff
@@ -261,7 +311,10 @@ export function Header() {
                     isSectionOpen(SECTION_TYPE.FINANCE) && 'tw-font-bold'
                   }`}
                 >
-                  <span className="tw-mr-2 2xl:tw-mr-3 tw-cursor-pointer">
+                  <span
+                    className="tw-mr-2 2xl:tw-mr-3 tw-cursor-pointer"
+                    data-action-id="headerMenu-finance-btn"
```
Comment (soulBit) [inline src/app/components/Header/index.tsx:316]:
```suggestion
                    data-action-id="header-link-finance"
```

File: src/app/components/Header/index.tsx [hunk 14]
```diff
@@ -276,6 +329,7 @@ export function Header() {
                       onClick={() => {
                         history.push('/stake');
                       }}
+                      data-action-id="bitocracyMenu-stake-btn"
```
Comment (soulBit) [inline src/app/components/Header/index.tsx:332]:
```suggestion
                      data-action-id="header-bitocracy-link-stake"
```

File: src/app/components/Header/index.tsx [hunk 15]
```diff
@@ -289,6 +343,7 @@ export function Header() {
                       target="_blank"
                       text={t(translations.mainMenu.governance)}
                       className="bp3-popover-dismiss"
+                      data-action-id="bitocracyMenu-governance-btn"
```
Comment (soulBit) [inline src/app/components/Header/index.tsx:346]:
```suggestion
                      data-action-id="header-bitocracy-link-governance"
```

File: src/app/components/Header/index.tsx [hunk 16]
```diff
@@ -302,6 +357,7 @@ export function Header() {
                       target="_blank"
                       text={t(translations.mainMenu.forum)}
                       className="bp3-popover-dismiss"
+                      data-action-id="bitocracyMenu-forum-btn"
```
Comment (soulBit) [inline src/app/components/Header/index.tsx:360]:
```suggestion
                      data-action-id="header-bitocracy-link-forum"
```

File: src/app/components/Header/index.tsx [hunk 17]
```diff
@@ -311,7 +367,10 @@ export function Header() {
                     isSectionOpen(SECTION_TYPE.BITOCRACY) && 'font-weight-bold'
                   }`}
                 >
-                  <span className="tw-mr-2 2xl:tw-mr-3 tw-cursor-pointer">
+                  <span
+                    className="tw-mr-2 2xl:tw-mr-3 tw-cursor-pointer"
+                    data-action-id="headerMenu-bitocracy-btn"
```
Comment (soulBit) [inline src/app/components/Header/index.tsx:372]:
```suggestion
                    data-action-id="header-link-bitocracy"
```

File: src/app/components/Header/index.tsx [hunk 18]
```diff
@@ -320,13 +379,15 @@ export function Header() {
               <NavLink
                 className="tw-header-link tw-mr-2 2xl:tw-mr-3"
                 to="/reward"
+                data-action-id="headerMenu-reward-btn"
```
Comment (soulBit) [inline src/app/components/Header/index.tsx:382]:
```suggestion
                data-action-id="header-link-rewards"
```

File: src/app/components/Header/index.tsx [hunk 19]
```diff
@@ -320,13 +379,15 @@ export function Header() {
...
               >
                 {t(translations.mainMenu.rewards)}
               </NavLink>
 
               <NavLink
                 className="tw-header-link tw-mr-2 2xl:tw-mr-3"
                 to="/wallet"
+                data-action-id="headerMenu-portfolio-btn"
```
Comment (soulBit) [inline src/app/components/Header/index.tsx:390]:
```suggestion
                data-action-id="header-link-portfolio"
```

File: src/app/components/Header/index.tsx [hunk 20]
```diff
@@ -335,6 +396,7 @@ export function Header() {
                 target="_blank"
                 rel="noopener noreferrer"
                 className="tw-header-link"
+                data-action-id="headerMenu-bridge-btn"
```
Comment (soulBit) [inline src/app/components/Header/index.tsx:399]:
```suggestion
                data-action-id="header-link-bridge"
```

File: src/app/components/Header/index.tsx [hunk 21]
```diff
@@ -345,11 +407,13 @@ export function Header() {
                       text={t(translations.mainMenu.launchpad)}
                       className="bp3-popover-dismiss"
                       onClick={() => history.push('/origins')}
+                      data-action-id="originsMenu-launchpad-btn"
```
Comment (soulBit) [inline src/app/components/Header/index.tsx:410]:
```suggestion
                      data-action-id="header-link-origins"
```

File: src/app/components/Header/index.tsx [hunk 22]
```diff
@@ -345,11 +407,13 @@ export function Header() {
...
                     />
                     <MenuItem
                       text={t(translations.mainMenu.claim)}
                       className="bp3-popover-dismiss"
                       onClick={() => history.push('/origins/claim')}
+                      data-action-id="originsMenu-claim-btn"
```
Comment (soulBit) [inline src/app/components/Header/index.tsx:416]:
```suggestion
                      data-action-id="header-origins-link-claim"
```

File: src/app/components/Header/index.tsx [hunk 23]
```diff
@@ -359,7 +423,10 @@ export function Header() {
                     isSectionOpen(SECTION_TYPE.ORIGINS) && 'tw-font-bold'
                   }`}
                 >
-                  <span className="tw-mr-2 2xl:tw-mr-3 tw-cursor-pointer">
+                  <span
+                    className="tw-mr-2 2xl:tw-mr-3 tw-cursor-pointer"
+                    data-action-id="headerMenu-origins-btn"
```
Comment (soulBit) [inline src/app/components/Header/index.tsx:428]:
```suggestion
                    data-action-id="header-link-origins"
```

File: src/app/components/Header/index.tsx (hunk 4 above)
Comment (tiltom) [reply src/app/components/Header/index.tsx]: thank you for the review :) I am going to finish the naming

**Review by soulBit — DISMISSED** [review]

**Pull request merged**